packages/react/package.json

```diff
@@ -78,17 +78,15 @@
   "dependencies": {
     "@clerk/shared": "workspace:*",
     "@clerk/types": "workspace:*",
-    "tslib": "2.4.1"
+    "tslib": "catalog:repo"
   },
   "devDependencies": {
     "@clerk/eslint-config-custom": "workspace:*",
     "@clerk/localizations": "workspace:*",
     "@clerk/themes": "workspace:*",
     "@types/node": "^18.19.33",
-    "@types/react": "18.3.12",
-    "@types/react-dom": "18.3.1",
```

Comment:
@jacekradko What's up with this? Why are you dropping types here, and what's the goal if you don't mind? Maybe I don't understand catalogs here.

I'm working on Next.js 15 + `@clerk/nextjs@6` support for Convex, we also have a pnpm monorepo (via Rush) and wondering if this is your approach to a pnpm repo with multiple React versions without adding your types to peerDependencies. Happy to chat in the Clerk/Convex Slack connect too.